projects/hipsparselt/library/CMakeLists.txt

```
@@ -46,6 +46,14 @@ set(hipsparselt_headers_public
 
 source_group("Header Files\\Public" FILES ${hipsparselt_headers_public})
 
+# work around code object stripping failure if using /usr/bin/strip
+set(CPACK_RPM_SPEC_MORE_DEFINE "%define __strip /opt/rocm/llvm/bin/llvm-strip") # need to generalize this
+install(
+    FILES ${hipbsparselt_public_headers}
```

```suggestion
    FILES ${hipsparselt_headers_public}
```
Typo. Also, should it be this?